### PR TITLE
Added examples for kn usage for the Ruby sample app

### PR DESCRIPTION
**Pull request by maximilien**
* Added examples for kn usage for the Ruby sample app

### CODE REVIEW

**Comment by tcnghia** [comment]
@maximilien can you please rebase?

File: docs/serving/samples/hello-world/helloworld-ruby/index.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+cd knative-docs/docs/serving/samples/hello-world/helloworld-ruby
+```
+
+## Prerequisites
+
+- A Kubernetes cluster with Knative installed and DNS configured. Follow the
+  [installation instructions](../../../../install/README.md).
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured.
+- (optional) The Knative CLI client [kn](https://github.com/knative/client/releases) that simplifies the deployment. Alternative you can also use [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and apply resource files directly.
```
Comment (mpetason) [inline docs/serving/samples/hello-world/helloworld-ruby/index.md:27]:
```suggestion
- (optional) The Knative CLI client [kn](https://github.com/knative/client/releases) that simplifies the deployment. Alternatively, you can also use [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and apply resource files directly.
```

**Comment by maximilien** [comment]
hi, @tcnghia and @mpetason I will rebase and address this. Sorry about delay.

**Comment by maximilien** [comment]
Thanks for all the feedback folks. I will try to get to this today and resubmit. Sorry about delays.

**Comment by abrennan89** [comment]
@maximilien are you still working on this one or is it ready for another review / merge? :slightly_smiling_face:

**Review by abrennan89 — APPROVED** [review]
/lgtm
YOLO :upside_down_face: looks like it's just moving from the README and is passing the important tests so I'll merge for now and if we need updates we can do them later

**Pull request merged**